packages/wrangler/src/pages/publish.tsx

```diff
@@ -354,6 +358,7 @@ export const Handler = async ({
 
 			_routes = JSON.stringify(optimizeRoutesJSONSpec(advancedModeRoutes));
 			formData.append("_routes.json", new File([_routes], "_routes.json"));
+			logger.log(`✨ Uploading _routes.json`);
```

Comment:
Do we want to log this? CC: @jahands

Reply:
I decided to do it because it's only uploaded when
a) the user has a _worker.js
b) the user manually provides a _routes.json with it
Additionally, on the line below the logger warns that it's an experimental feature, however if it then doesn't say like it's uploading it, compared to the others, it feels like it's not, and therefore is broken

Reply:
Happy to remove it if desired though

Reply:
That makes sense to me

Reply:
I like this - thanks!

Reply:
It is annoying wondering if these magic files are actually getting applied, so I really like it logging them out like this